components/tools/OmeroJava/test/integration/thumbnail/SkipThumbnailsPermissionsTest.java

```diff
@@ -260,22 +308,91 @@ public void testGetThumbnailWithRenderingSettingsChange(String permissions, bool
         return testCases.toArray(new Object[testCases.size()][]);
     }
 
+    private EventContext addUser(EventContext user1, boolean isAdmin, boolean isGroupOwner) throws Exception {
+        // Create new user in group and login as that user
+        EventContext newUser = newUserInGroup(user1, isGroupOwner);
+        if (isAdmin) {
+            // If user is an admin, add them to the system group
+            ExperimenterGroup systemGroup = new ExperimenterGroupI(iAdmin.getSecurityRoles().systemGroupId, false);
+            addUsers(systemGroup, Collections.singletonList(newUser.userId), false);
+        }
+        return newUser;
+    }
+
+    private EventContext addUserAndLogin(EventContext user1, boolean isAdmin, boolean isGroupOwner) throws Exception {
+        // Login as new user
+        EventContext newUser = addUser(user1, isAdmin, isGroupOwner);
+        loginUser(newUser);
+        return newUser;
+    }
+
     private Pixels importFile(ImportConfig config) throws Throwable {
         String format = ModelMockFactory.FORMATS[0];
         File file = createImageFile(format);
         return importFile(config, file, format).get(0);
     }
 
     private Pixels importLargeFile(ImportConfig config) throws Throwable {
-        String format = ModelMockFactory.FORMATS[0];
-        File file = createImageFile(format);
-        return importFile(config, file, format).get(0);
+        BufferedImage bi = new BufferedImage(4096, 4096, BufferedImage.TYPE_INT_RGB);
```

Comment:
Perhaps an empty `big-image&sizeX=4096&sizeY=4096.fake` file would suffice?

Reply:
it will